### PR TITLE
Re-add picard createorreplacereadgroups for MultiVCFanalyzer

### PR DESCRIPTION
**Pull request by jfy133**
I now remember why we had it in the first place :roll_eyes: 

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](<https://github.com/>nf-core/eager/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/eager _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by jfy133** [comment]
@meganemichel and @aidaanva please try this branch `-r patch-multivcfanalyzer-readgroups` on your data both with and without running MultiVCFAnalyzer and check the VCF files look good (without MVA: include multiple samples in the VCF; with MVA: single samples in VCF so compatible with MVA)

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Changed endors.py license from GPL to MIT (by @aidaanva)
 - Removed erroneous R2 in single-end example in input TSV of usage docs (by @aidaanva)
+- [#928](https://github.com/nf-core/eager/issues/928) Fixed read group incompatibility for MultiVCFAnalyzer (♥ to @aidaanva, @meganemichel for reporting)
```
Comment (jfy133) [inline CHANGELOG.md:14]:
```suggestion
- [#928](https://github.com/nf-core/eager/issues/928) Fixed read group incompatibility by re-adding picard createorreplacereadgroups for MultiVCFAnalyzer (♥ to @aidaanva, @meganemichel for reporting)
```

**Comment by TCLamnidis** [comment]
isnt the tool called AddOrReplaceReadGroups?

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Changed endors.py license from GPL to MIT (by @aidaanva)
 - Removed erroneous R2 in single-end example in input TSV of usage docs (by @aidaanva)
+- [#928](https://github.com/nf-core/eager/issues/928) Fixed read group incompatibility by re-adding picard createorreplacereadgroups for MultiVCFAnalyzer (♥ to @aidaanva, @meganemichel for reporting)
```
Comment (jfy133) [inline CHANGELOG.md:14]:
```suggestion
- [#928](https://github.com/nf-core/eager/issues/928) Fixed read group incompatibility by re-adding picard AddOrReplaceReadGroups for MultiVCFAnalyzer (♥ to @aidaanva, @meganemichel for reporting)
```

**Comment by meganemichel** [comment]
Tested patch and confirm that running with  `--run_multivcfanalyzer` triggers updating of read groups, while running without `--run_multivcfanalyzer` produces unmodified vcfs.

**Review by apeltzer — APPROVED** [review]
Yes, 🥇

**Pull request merged**